core/benchmarks/test/src/fr/hammons/slinc/BindingsBenchmarkShape.scala

```diff
@@ -68,21 +75,23 @@ trait BindingsBenchmarkShape(val s: Slinc):
 
   @Benchmark
   @OutputTimeUnit(TimeUnit.MILLISECONDS)
-  def qsort =
-    for size <- SizeT.maybe(10000)
-    do
-      Scope.confined {
-        val sortingArr = Ptr.copy(baseArr).castTo[Nothing]
-        cstd.qsort(
-          sortingArr,
-          size,
-          IntDescriptor.size.toSizeT,
-          upcall
-        )
-        sortingArr.castTo[Int].asArray(10000)
-      }
+  def qsort(state: QSortState, bh: Blackhole): Unit =
+    bh.consume {
+      for size <- SizeT.maybe(10000)
```

Comment:
perhaps, should we put `SizeT.maybe(10000)` outside benchmark iterations?

Reply:
I'm specifically wanting to include the instantiation of platform dependent types in the benchmark, just to see if they add much or not (and to be a fairer comparison). If you'd like, maybe make a second qsort bench with the SizeT cached?  Also, `SizeT(10000: Short)` should be weightless to the point of being a no-op (just tagging 10000 with the SizeT type for the compiler, optimized away by the JVM), so maybe you could create a third bench just based on that?